### DISCUSSION TITLE
Loan Amount in Sole Prop

### DISCUSSION THREAD

**Discussion by ravi09304** [ONDC-Official/ONDC-FIS-Specifications#34] [General]
I remember from the initial call that loan amount of sole_prop was based on GSTIN data and it was a revolving limit proposition, in the figma screen the loan_amount shown is of the Invoice value. 

So, how to convey customer that the total limit is different from invoice value (loan amount) and will the customer be shown the overall limit if the customer wants another Invoice discounting post successful onboarding.

**Comment by Antriksh-Parmar** [comment]
@ravi09304 The MSME use case is essentially a one-time unsecured credit against the cash flows. The value of the offered loan amount would slightly lower than the invoice value.

**Reply by ravi09304** [reply]
@Antriksh-Parmar  So for a new Invoice discounting, it will be treated as new loan with the approximate amount of Invoice value? 

Also, here if the borrower is already onboarded with the lender, then what will be the flow? Just the loan approval process?

**Reply by Antriksh-Parmar** [reply]
It's not invoice discounting per se. The loan is provided against cashflow, but the invoices are not pledged.
But yes, a certain percentage of the invoice amount will be extended as a loan, let's say anything between 60-85%, just a ballpark.
Assuming you're referring to existing customers, the process will remain the same, minus the KYC steps.
